fdbserver/storageserver.actor.cpp

```diff
@@ -8615,7 +8622,7 @@ ACTOR Future<Void> update(StorageServer* data, bool* pReceivedUpdate) {
 				    "A fetchKeys completed while we were doing this, so eager might be outdated.  Read it again.");
 				// SOMEDAY: Theoretically we could check the change counters of individual shards and retry the reads
 				// only selectively
-				eager = UpdateEagerReadInfo();
+				eager = UpdateEagerReadInfo(enableClearRangeEagerReads);
```

Comment:
It would be a bit cleaner to just make a `.clear()` method on `UpdateEagerReadInfo` rather than each creation point passing the same args.  Operator= is only used here as a way of clearing the structure.